rfcs/007 - Peer review infra changes.md

```diff
@@ -31,23 +31,23 @@ Eng teams move over from using cloud UIs to IaC because cloud infrastructure cha
 - Allow plain SQL or have an introspective ORM
 - Free and ideally open source
 - Baseline/ignore certain migrations
-- Be resilient to `iasql_ugprade`
+- Be resilient to IaSQL schema upgrades which involve tables getting dropped and recreated
```

Comment:
We actually don't drop tables anymore during upgrade, we create a new database and then swap to it.